src/dataExplorer/context/tags.tsx

```diff
@@ -110,19 +110,21 @@ export const TagsProvider: FC<Prop> = ({children, scope}) => {
     `
 
     if (bucket.type !== 'sample' && isFlagEnabled('newQueryBuilder')) {
-      _source = `${IMPORT_REGEXP}${IMPORT_INFLUX_SCHEMA}`
+      _source = `${IMPORT_STRINGS}${IMPORT_INFLUX_SCHEMA}`
       queryText = `${_source}
         schema.measurementTagKeys(
           bucket: "${bucket.name}",
           measurement: "${measurement}",
           start: ${CACHING_REQUIRED_START_DATE},
           stop: ${CACHING_REQUIRED_END_DATE},
         )
-          |> filter(fn: (r) => r._value != "_measurement" and r._value != "_field")
-          |> filter(fn: (r) => r._value != "_start" and r._value != "_stop")
-          ${searchTerm ? SEARCH_STRING(searchTerm) : ''}
-          |> sort()
-          |> limit(n: ${limit})
+        |> filter(fn: (r) => r._value != "_measurement" and r._value != "_field")
+        |> filter(fn: (r) => r._value != "_start" and r._value != "_stop")
+        ${searchTerm ? SEARCH_STRING(searchTerm) : ''}
```

Comment:
dont we still need the regex imported for `SEARCH_STRING`?

Reply:
yes, since we are using strings package `strings.toLower()` in the next line